src/server.py

```diff
@@ -370,8 +370,16 @@ def ref(self, uuid_, bus_name):
         self._refcount[uuid_][bus_name] += 1
 
     def unref(self, uuid_, bus_name, count=1):
+        msg_args = (uuid_, bus_name)
+        if uuid_ not in self._refcount:
+            _logger.warning("{}: This uuid is not registered in the refcount "
+                            "registry.".format(uuid_))
+            return
+        if bus_name not in self._refcount[uuid_]:
+            _logger.warning("{}: Bus name '{}' is not registered in the "
+                            "refcount registry.".format(*msg_args))
```

Comment:
I think it's more readable and safer to use `format(uuid_, bus_name)` here, otherwise if the tuple above is changed, this breaks (silently until an error happens).